guacamole/src/main/webapp/app/form/templates/timeZoneField.html

```diff
@@ -3,12 +3,14 @@
     <!-- Available time zone regions -->
     <select class="time-zone-region"
             ng-disabled="disabled"
+            guac-focus="focused"   
             ng-model="region"
             ng-options="name for name in regions | orderBy: name"></select>
 
     <!-- Time zones within selected region -->
     <select class="time-zone"
             ng-disabled="disabled || !region"
+            guac-focus="focused"
```

Comment:
Having `guac-focus="focused"` for both `<select>` fields doesn't make sense, as you can only have one field focused at a time. What is the intended effect?

Reply:
What you point out is indeed correct and there was no legit reasoning behind this besides the fact that I was applying this `guac-focus` on every field template out there. I will make appropriate changes based on actual reasoning this time around.

Reply:
The new approach that I believe is suitable for this field is to have the `guac-focus` directive applied only to the "time-zone-region" and there are a few reasons why I believe it is the right to do.
1. In most cases, it is impossible to determine which part of the field is relevant so it makes sense to naturally focus the beginning of this field.
2. Also, the "time-zone" field is disabled until some content is selected in the "time-zone-region". Because of this dependency, I think it is best to focus the field that does not depend on the other.
3. And last but not the least, it is simple.

Reply:
Sure. Focusing just the first `<select>` makes sense to me.